OIPA/.pytest_cache/README.md

```diff
@@ -0,0 +1,8 @@
+# pytest cache directory #
```

Comment:
I can see that you accidentally added `.pytest_cache/` directory to the repo haha :)